QtQuickApplication/TabGeneral.qml

```diff
@@ -156,30 +249,16 @@ Rectangle{
 
             Component.onCompleted: initCustomPlot()
 
-            /*
             Timer {
```

Comment:
maybe we can use just one time in the whole qml, we can check this later. Pay attention because a time will just get started when the qml gets shown.